bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/src/main/java/com/google/cloud/bigtable/mapreduce/validation/SyncTableValidationOnlyJob.java

```diff
@@ -0,0 +1,1102 @@
+    System.err.println(" Or Source Configuration:");
+    System.err.println("    sourcebigtableproject  Bigtable project id of the source table");
+    System.err.println("                      (defaults to cluster in classpath's config)");
+    System.err.println("    sourcebigtableinstance  Bigtable instance id of the source table");
+    System.err.println("                      (defaults to cluster in classpath's config)");
+    System.err.println("    sourcebigtableprofile  (optional) Bigtable app profile");
+    System.err.println(" Target Configuration:");
+    System.err.println("    targetzkcluster  ZK cluster key of the target table");
+    System.err.println("                      (defaults to cluster in classpath's config)");
+    System.err.println(" Or Target Configuration:");
+    System.err.println("    targetbigtableproject  Bigtable project id of the target table");
+    System.err.println("                      (defaults to cluster in classpath's config)");
+    System.err.println("    targetbigtableinstance  Bigtable instance id of the target table");
+    System.err.println("                      (defaults to cluster in classpath's config)");
+    System.err.println("    targetbigtableprofile  (optional) Bigtable app profile");
+    System.err.println();
+    System.err.println(" sourcehashdir     path to HashTable output dir for source table");
+    System.err.println("                   (see org.apache.hadoop.hbase.mapreduce.HashTable)");
+    System.err.println(" sourcetablename   Name of the source table to sync from");
+    System.err.println(" targettablename   Name of the target table to sync to");
+    System.err.println(" outputpath        Filesystem path to put the sync output data");
+    System.err.println();
+    System.err.println("Examples:");
+    System.err.println(" For SyncTable validation of tableA from a remote source cluster");
+    System.err.println(" to a local target cluster:");
+    System.err.println(
+        " $ bin/hbase "
+            + "com.google.cloud.bigtable.mapreduce.validation.SyncTable --targetbigtableproject=project123"
+            + " --targetbigtableinstance=instance123 --sourcezkcluster=zk1.example.com,zk2.example.com,zk3.example.com:2181:/hbase"
+            + " gs://bucket/hashes/tableA tableA tableA");
+  }
+
+  @Override
+  public int run(String[] args) throws Exception {
+    String[] otherArgs = new GenericOptionsParser(getConf(), args).getRemainingArgs();
+    if (!doCommandLine(otherArgs)) {
+      return 1;
+    }
+
+    Job job = createSubmittableJob(otherArgs);
+    if (!job.waitForCompletion(true)) {
+      return 1;
+    }
+    counters = job.getCounters();
+    return 0;
+  }
+
+  protected boolean doCommandLine(String[] args) {
+    if (!parseCommandLine(args)) return false;
+    if (!verifyRequiredArgs(args)) return false;
+    return true;
+  }
+
+  private boolean verifyRequiredArgs(String[] args) {
```

Comment:
Why is targetZKCluster or targetBigtableProject/Instance not required? I am assuming target is the migration target where the new data is going. If at all source should be optional and not target?